OpenTabletDriver.Desktop/Output/LinuxArtistMode.cs

```diff
@@ -26,6 +26,11 @@ protected override void OnOutput(IDeviceReport report)
 
             if (report is ITiltReport tiltReport)
                 VirtualTablet.SetTilt(tiltReport.Tilt);
+
+            VirtualTablet.SetEraser(report is IEraserReport); // FIXME: this is probably inefficient
```

Comment:
`IEraserReport` just states that it reports whether or not the eraser is reporting pressure.
```suggestion
            if (report is IEraserReport eraserReport)
                VirtualTablet.SetEraser(eraserReport.Eraser);
```

Reply:
Are you sure? Looking at report parsers, it seems that it's set whenever you've got the eraser end of the pen in action, and not just when pressure is present.
I can check tomorrow whether a G960S+ reports eraser state in hover or just on pressure as you said - I have no other tablets to test this.

Your change is however correct and does what the intention was

Reply:
> Are you sure? Looking at report parsers, it seems that it's set whenever you've got the eraser end of the pen in action, and not just when pressure is present.

That's what I meant, 0 pressure is still reporting pressure.